### PR TITLE
Support reading wflow models with input and/or output directories in the toml

### PR DESCRIPTION
**Pull request by JoostBuitink**
Last year, the option to specify a `dir_input` and or `dir_output` was added to Wflow.jl. However, these options were not know to hydromt_wflow, so reading the model would skip the input and/or output files when this is specified in the Wflow .toml file. These changes fix this, and also adds the `dir_output` to the default toml file to improve the user experience of Wflow.

### CODE REVIEW

**Review by hboisgon — CHANGES_REQUESTED** [review]
Nice additions @JoostBuitink , thanks a lot!
I think really nice to change the default dir_output options as well: this however means that the built models will contain the change so good to check if the examples need to be updated and document the change in the changelog.

I also have a suggestion on how to insert dir_input and dir_output and suggest to apply the abs_path argument of get_config for them instead of the filename directly, might make it more robust.

**Comment by hboisgon** [comment]
Hi @JoostBuitink 
Thanks for the changes! I left a small suggestion with the review :)

I see the tests are failing on glaciers though... not sure why... could you check it or do you prefer if I do?

File: hydromt_wflow/wflow.py [hunk 1]
```diff
@@ -1698,6 +1698,12 @@ def read_staticmaps(self, **kwargs):
         """Read staticmaps"""
         fn_default = join(self.root, "staticmaps.nc")
         fn = self.get_config("input.path_static", abs_path=True, fallback=fn_default)
+
+        if "dir_input" in self.config.keys():
```
[the remark this answers is not in the document]
Comment (JoostBuitink) [reply hydromt_wflow/wflow.py]: Good suggestion, thanks! I have implemented this now (also for all other checks for `dir_input` and `dir_output`)

**Comment by JoostBuitink** [comment]
Hi @hboisgon! Thanks for the quick review! I have included your changes, and am currently looking in to the failing glacier test. Will let you know if I need some help with that!

**Comment by JoostBuitink** [comment]
The failing test with the glaciers appears to be unrelated to this PR, as it also happend in another PR. I have created https://github.com/Deltares/hydromt_wflow/issues/143 to cover this problem

**Review by hboisgon — APPROVED** [review]

**Pull request merged**